pkg/remote/aws/test/dynamodb_table_list/results.golden.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
We expect two tables there ⚠️

Reply:
no we don't it's the test for enumeration (list) error ^^

Reply:
I could rename the test directory but I used the same naming for all the listing test...

Reply:
Yeah the naming is pretty confusing but we'll refacto this in another PR